### PR TITLE
Loop over colour scheme

### PR DESCRIPTION
**Pull request by pollen8**
an error was thrown if we had more data than colour scheme items. 
This fixes the issue by looping over the scheme with a modulus

### CODE REVIEW

File: src/utils/colorScheme.tsx [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { scaleOrdinal } from "d3-scale";
```
Comment (stusmitharoo) [inline src/utils/colorScheme.tsx:1]: Is this an unused var? I can see its been added but not where it has been used.

File: src/components/Bars/barHelper.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import { ScaleBand, ScaleLinear } from "d3-scale";
+import { ScaleBand, ScaleLinear, scaleOrdinal } from "d3-scale";
```
Comment (stusmitharoo) [inline src/components/Bars/barHelper.ts:1]: Is this an unused var? I can see its been added but not where it has been used.

Comment (pollen8) [reply src/components/Bars/barHelper.ts]: Yes its unused, thought I needed it then I didn't! Thanks - updated

File: src/utils/colorScheme.tsx (hunk 1 above)
Comment (pollen8) [reply src/utils/colorScheme.tsx]: Yes its unused, thought I needed it then I didn't! Thanks - updated

**Review by stusmitharoo — APPROVED** [review]

**Pull request merged**